dev/com.ibm.websphere.appserver.features/visibility/public/mpHealth-3.0/resources/l10n/io.openliberty.mpHealth-3.0_de.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=Dieses Feature stellt Unterst\u00fctzung f\u00fcr MicroProfile Health API f\u00fcr Java EE (Java Platform, Enterprise Edition) bereit.
```

Comment:
My German might not be perfect but this text does not look the same as the english? I think the same applies to the other translated files here.

Reply:
I removed all the translated files and only kept the English file, so it can get translated again properly.